core/src/main/resources/hudson/model/Messages_fr.properties

```diff
@@ -21,62 +21,127 @@
 # THE SOFTWARE.
 
 AbstractBuild.BuildingRemotely=Construction à distance sur {0}
+AbstractBuild.BuildingOnMaster=Construction sur le nœud contrôleur
+AbstractBuild_Building=En cours de construction
+AbstractBuild.BuildingInWorkspace=\ dans le répertoire de travail {0}
 AbstractBuild.KeptBecause=conservé à cause de {0}
 
-AbstractProject.NewBuildForWorkspace=Demande d''un nouveau build afin d''avoir un répertoire de travail
+AbstractItem.NoSuchJobExists=Job ‘{0}’ inexistant. Vouliez-vous ‘{1}’?
+AbstractItem.NoSuchJobExistsWithoutSuggestion=Job ‘{0}’ inexistant.
+AbstractItem.Pronoun=Item
+AbstractItem.TaskNoun=Build
+AbstractItem.BeingDeleted={0} en cours de suppression
+AbstractItem.FailureToStopBuilds=Echec de l''interruption et de l''arrêt de {0,choice,1#{0,number,integer} build|1<{0,number,integer} \
+  builds} de {1}
+AbstractItem.NewNameInUse=Le nom “{0}” est déjà utilisé.
```

Comment:
@lemeurherve 
Mmmh, not sure about this one, as the default [Messages.properties](https://github.com/jgreffe/jenkins/blob/2fc718da2a377d9dc411ebae95badf2f42212bef/core/src/main/resources/hudson/model/Messages.properties#L38) uses specific quotes.

Reply:
I'm not sure why there are two sort of double quotes characters in the beginning, I think keeping only the standard ones would be better.

Question for the usual maintainers, what is the consensus (if any) about that?

Same about the use of special single quotes like L43 & L44 (`‘{0}’`) instead of double quotes?
Is it intentional or some sort of crust?

Note: these conversions could also be done in another PR, absolutely not blocking for this one.